### DISCUSSION TITLE
Security Onion preflight fail

### DISCUSSION THREAD

**Discussion by rhoadrw** [Security-Onion-Solutions/securityonion#8068] [Unsupported Versions] [answered]
I am installing the latest version of Security Onion.  The pre-flight failed on the raw.gethubusercontent.com check because we use Umbrella and the certificate check is seeing that and failing.  Is there a way to bypass these checks or otherwise force the install to happen?

**Comment by weslambert** [comment, marked as answer]
It sounds like you're going to end up having more problems than just that if you are already having issues with certificate validation/verification with preflight only. You might consider altering proxy settings (https://docs.securityonion.net/en/latest/proxy.html), or allowing Security Onion to bypass Umbrella, or perform an airgap (https://docs.securityonion.net/en/latest/airgap.html) install (keep in mind, you would need to manually update with an updated ISO image each time).

Thanks,
Wes

**Comment by rhoadrw** [comment]
Thank you.  I was afraid of those answers you have given.  I have caved, and exempted the Security Onion systems from enforced use of Umbrella.
